sdk/communication/Azure.Communication.PhoneNumbers/tests/PhoneNumbersClient/PhoneNumbersClientLiveTests.cs

```diff
@@ -175,15 +175,29 @@ public async Task UpdateCapabilitiesAsync()
             var number = GetTestPhoneNumber();
 
             var client = CreateClient();
-            var updateOperation = await client.StartUpdateCapabilitiesAsync(number, PhoneNumberCapabilityType.Outbound, PhoneNumberCapabilityType.Outbound);
+            var phoneNumber = await client.GetPurchasedPhoneNumberAsync(number);
+            PhoneNumberCapabilityType callingCapabilityType = PhoneNumberCapabilityType.Outbound;
+            PhoneNumberCapabilityType smsCapabilityType = PhoneNumberCapabilityType.InboundOutbound;
+
+            if (phoneNumber.Value.Capabilities.Calling == callingCapabilityType)
```

Comment:
Why do we swap instead of going back to the original state at the end of the test?

Reply:
I dont think we need to go back to original state because all we need to guarantee here is that we are updating capabilities to a different state. In addition, updating back to original state may actually cause a race condition because we are doing two back to back updates, which is what we were trying to avoid in the first place.